src/apriltag_publisher.cpp

```diff
@@ -0,0 +1,85 @@
+#include <string>
+
+#include <ros/ros.h>
+#include <tf/transform_listener.h>
+#include <tf2_ros/transform_listener.h>
+#include <geometry_msgs/TransformStamped.h>
+#include "apriltag_ros/AprilTagDetectionArray.h"
+#include "apriltag_ros/AprilTagDetection.h"
+
+// Agent info
+const std::vector<std::string> agent_names = {"agent1", "agent2"};
+const std::string base_frame_name = "base_link";
+// Tag info
+const std::string tag_name = "tag_";
+const size_t tag_amount = 20;
+// Topic info
+const std::string topic_name_prefix = "/robosar_agent_bringup_node/";
+const std::string topic_name_suffix = "/feedback/apriltag";
```

Comment:
Can we make these rosparams? Bc I will also have to use them on slam_toolbox side to subscribe

Reply:
Done